DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/handshake-control/Readme.md

```diff
@@ -0,0 +1,199 @@
+# Streaming Invocation and Conduit Arguments
+This implementation uses a streaming invocation interface and conduits to pass the input pointer arguments instead of the CSR. It uses a single, shared memory-mapped host data interface.
+This design will show how to specify conduit interfaces using `annotated_arg`.
+
+![](../assets/stream_kernel.svg)
+
+## Invocation Interface - Streaming pipelined interface
+In this design, the `SimpleVAdd` kernel uses a streaming pipelined invocation interface. This is explicitly specified using kernel properties. 
+
+In this design following member functions is added to the kernel functor to specify kernel properties:
+```cpp
+auto get(sycl::ext::oneapi::experimental::properties_tag) {
+    return sycl::ext::oneapi::experimental::properties{
+        sycl::ext::intel::experimental::
+            streaming_interface<>,
```

Comment:
strange formatting here, is that how clang-format auto formats it?

Reply:
that looks different than the source code, I probably hit return by accident :(